Clarify that issuing `feature` via the commandline requires 'accept' or 'reject' booleans. (#1088)
XRPLF/xrpl-dev-portal: content/references/rippled-api/admin-rippled-methods/status-and-debugging-methods/feature.md

| `Field`   | Type    | Description                                            |
|:----------|:--------|:-------------------------------------------------------|
| `feature` | String  | _(Optional)_ The unique ID of an amendment, as hexadecimal; or the short name of the amendment. If provided, limits the response to one amendment. Otherwise, the response lists all amendments. |

> | `vetoed`  | Boolean | (Optional; ignored unless `feature` also specified) If true, instructs the server to vote against the amendment specified by `feature`. If false, instructs the server to vote in favor of the amendment. Use 'accept' or 'reject rather than 'true' or 'false' when using the commandline. |

Reviewer: accept

> | `vetoed`  | Boolean | (Optional; ignored unless `feature` also specified) If true, instructs the server to vote against the amendment specified by `feature`. If false, instructs the server to vote in favor of the amendment. Use 'accept' or 'reject rather than 'true' or 'false' when using the commandline. |

Reviewer 2: ```suggestion
| `vetoed`  | Boolean | (Optional; ignored unless `feature` also specified) If true, instructs the server to vote against the amendment specified by `feature`. If false, instructs the server to vote in favor of the amendment. On the commandline, use 'accept' or 'reject rather than 'true' or 'false'. |
```
